src/Components/Layout/resizer.ts

```diff
@@ -1,78 +1,75 @@
 import Storage from '../../Api/storage';
-/**
- * Sidebar resizer initalization
- * @returns {Promise<void>}
- */
-const Resizer = async (): Promise<void> => {
-	const sidebarResizer = document.getElementById('sidebar-resizer');
-	const sidebar = document.querySelector<HTMLElement>('.sidebar');
-	const settingsSidebar = document.querySelector<HTMLElement>('.settings-sidebar');
-	const xplorerBrand = document.querySelector<HTMLElement>('.xplorer-brand');
-	const appearance = await Storage.get('appearance');
-	const size = appearance?.sidebarWidth ?? '250px';
-	sidebar.style.flexBasis = size;
-	settingsSidebar.style.flexBasis = size;
 
-	let sidebarText: HTMLElement;
-	document.body.addEventListener('mousemove', (e: MouseEvent) => {
-		if (sidebar.classList.contains('sidebar-minimized')) {
-			if (
-				(e.target as HTMLElement).classList.contains('sidebar-item') ||
-				((e.target as HTMLElement).parentNode as HTMLElement).classList.contains('sidebar-item') ||
-				(e.target as HTMLElement).classList.contains('drive-item') ||
-				((e.target as HTMLElement).parentNode as HTMLElement).classList.contains('drive-item')
-			) {
-				let el = e.target as HTMLElement;
-				if (!el.classList.contains('drive-item') && !el.classList.contains('sidebar-item')) {
-					el = el.parentElement as HTMLElement;
-				}
-				sidebarText = el.querySelector('.sidebar-text') as HTMLElement;
-				const y = e.clientY - parseInt(getComputedStyle(el).getPropertyValue('padding-top'));
-				sidebarText.style.left = `70px`;
-				sidebarText.style.top = `${y}px`;
-			}
-		}
-	});
+const SENSOR = 10;
+const MIN_SIZE = 70;
+const MIN_SNAP = 220;
+
+let sidebar: HTMLElement;
+let settingsSidebar: HTMLElement;
+let xplorerBrand: HTMLElement;
+let appearance: any;
```

Comment:
I would like to assign `type Appearance` like the returned type of `Storage.get('appearance')` as on *line 40* to `appearance` but I don't know where to refer to.